clang/lib/Sema/SemaTemplateInstantiateDecl.cpp

```diff
@@ -5479,6 +5479,7 @@ void Sema::InstantiateVariableInitializer(
         *this, Sema::ExpressionEvaluationContext::PotentiallyEvaluated, Var);
 
     keepInLifetimeExtendingContext();
+    keepInRebuildDefaultArgOrInitContext();
```

Comment:
I am not sure that setting the flag in a one line function is better than doing it directly there

Reply:
I removed these 2 help functions, and doing it directly. It looks more clear.